journal/service.go

```diff
@@ -228,11 +227,13 @@ func toClientEvent(journal Journal) (clientEvent, error) {
 		return clientEvent{}, err
 	}
 
-	createdAtStr := journal.Attributes["created_at"].(string)
-	if createdAtStr != "" {
-		createdAt, err = time.Parse(time.RFC3339, createdAtStr)
-		if err != nil {
-			return clientEvent{}, fmt.Errorf("invalid created_at format")
+	if isCreate {
+		createdAtStr := journal.Attributes["created_at"].(string)
+		if createdAtStr != "" {
+			createdAt, err = time.Parse(time.RFC3339, createdAtStr)
+			if err != nil {
+				return clientEvent{}, fmt.Errorf("invalid created_at format")
+			}
 		}
```

Comment:
@felixgateru @dborovcanin 
What do you think about this approach
If we return error then it will again queue back. So we can log them or store in separate place 
```suggestion
	if isCreate {
		createdAtIf, ok := journal.Attributes["created_at"]
		if ok {
			createdAtStr, ok := createdAtIf.(string)
			if ok && createdAtStr != "" {
				createdAt, err = time.Parse(time.RFC3339, createdAtStr)
				if err != nil {
					fmt.Errorf("log error : invalid created_at format")
				}
			}
		}
```

Reply:
I don't mind having a trash to store unsuccessfully processed events.

Reply:
Yeah I agree, Can we do it in next PR ?

Reply:
Yes, this should be done in a separate PR. @felixgateru Please open a ticket for this.

Reply:
Done :https://github.com/absmach/supermq/issues/2880